packages/react-tooltip/src/components/Tooltip/Tooltip.types.ts

```diff
@@ -117,14 +101,18 @@ export type OnVisibleChangeData = {
  * Properties for Tooltip
  */
 export type TooltipProps = ComponentProps<TooltipSlots> &
-  Partial<Omit<TooltipCommons, 'content' | 'relationship'>> &
-  Pick<TooltipCommons, 'content' | 'relationship'>;
+  Partial<Omit<TooltipCommons, 'relationship'>> &
+  Pick<TooltipCommons, 'relationship'> & {
+    children?: (React.ReactElement & { ref?: React.Ref<unknown> }) | ((props: TooltipTriggerProps) => React.ReactNode);
```

Comment:
Previously `children` also had `| null`. Is there a reason you're removing it here?

```suggestion
    children?: (React.ReactElement & { ref?: React.Ref<unknown> }) | ((props: TooltipTriggerProps) => React.ReactNode) | null;
```

Reply:
No reason, applied the suggestion